EmotiBitBluetooth.cpp

```
@@ -0,0 +1,226 @@
+{
+        server -> deviceConnected = false;
+        Serial.println("BLE client disconnected");
+        //need to restart advertising to allow new connections after disconnection if accidentally disconnected
+        if (server -> _bluetoothReconnect)
+        {
+                server -> reconnect();
+                Serial.println("Restarted BLE advertising");
+        }
+}
+
+void EmotiBitBluetooth::MyCallbacks::onWrite(BLECharacteristic *pCharacteristic) 
```

Cppcheck (2.20.0)

[style] The function 'onWrite' is never used.

(unusedFunction)